### PR TITLE
Numerous small fixes

### PR DESCRIPTION
**Pull request by FinalAngel**
- Expand / Collapse added a "#" to the url
- Navigation can be again closed once opened when clicking on the root entry
- Aligned installation screen background with djangocms-admin-style
- Set correct paddings / margin between each navigation entry (between breaklines...)

### CODE REVIEW

File: cms/static/cms/js/modules/cms.toolbar.js [hunk 1]
```diff
@@ -149,6 +150,12 @@ var CMS = window.CMS || {};
                         e.stopPropagation();
                         var el = $(this);
 
+                        // close navigation once it's pressed again
+                        if (el.parent().hasClass(root) && open) {
+                            that.ui.body.trigger(that.click);
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.toolbar.js]: is it the same as calling `reset()` ?

File: cms/static/cms/sass/components/_toolbar.scss [hunk 2]
```diff
@@ -179,7 +179,7 @@
         @include hide-content();
         height: 1px;
         margin: 0 0 4px;
-        padding: 0 0 3px;
+        padding: 0 0 4px;
```
Comment (vxsx) [inline cms/static/cms/sass/components/_toolbar.scss]: :D

File: cms/static/cms/js/modules/cms.toolbar.js (hunk 1 above)
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.toolbar.js]: apparently not, calling reset has some weird effects. There might be other events also intertwined

**Comment by vxsx** [comment]
:+1:

**Pull request merged**